caffe-grid/src/main/python/com/yahoo/ml/caffe/CaffeOnSpark.py

```diff
@@ -51,5 +51,5 @@ def trainWithValidation(self,train_validation_source):
 
         :param DataSource: A list containing the source for training and validation data
         """
-        self.__dict__.get('cos').trainWithValidation(train_validation_source)
+        return self.__dict__.get('cos').trainWithValidation(train_validation_source)
```

Comment:
Yes will do it. Just pushed to check Travis error . Pull request is
incomplete

On Saturday, August 20, 2016, anfeng <notifications@github.com> wrote:

> In caffe-grid/src/main/python/com/yahoo/ml/caffe/CaffeOnSpark.py
> <https://github.com/yahoo/CaffeOnSpark/pull/135#discussion_r75587705>:
>
> > @@ -51,5 +51,5 @@ def trainWithValidation(self,train_validation_source):
> >
> >          :param DataSource: A list containing the source for training and validation data
> >          """
> > -        self.__dict__.get('cos').trainWithValidation(train_validation_source)
> > +        return self.__dict__.get('cos').trainWithValidation(train_validation_source)
>
> I thought that we will avoid array, and call out trainSrc and
> validationSrc explicitly.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/yahoo/CaffeOnSpark/pull/135/files/cb9786af4569a868ca2a6817297e45d2b2df999c#r75587705>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACCTVS5xoi6tyOF4gSgPl-ynf1gQopmTks5qh63ygaJpZM4JpLru>
> .
>